release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,15 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ptp-automatic-leap-seconds-handling-for-ptp-clocks_{context}"]
+==== Automatic leap seconds handling for PTP clocks
+
+You can now configure the PTP Operator to automatically update the leap second file by using Global Positioning System (GPS) announcements.
```

Comment:
I would say that PTP operator is configured by default to automatically ... (etc).